plugins/mcreator-core/jsontriggers/entity_hurt_player.json

```diff
@@ -0,0 +1,52 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "amount_l",
+      "check": "Number"
+    },
+    {
+      "type": "input_value",
+      "name": "amount_h",
+      "check": "Number"
+    },
+    {
+      "type": "field_input",
+      "name": "source_entity",
+      "text": "minecraft:zombie"
+    },
+    {
+      "type": "field_dropdown",
+      "name": "blocked",
+      "options": [
```

Comment:
Checkboxes (field_checkbox) are exactly doing this :)
https://developers.google.com/blockly/guides/create-custom-blocks/fields/built-in-fields/checkbox?hl=fr

Reply:
Updated!, 
```
i did see this mistake but the first time i tried it, it stored the value in Full Caps due to it trying to save it as a String, but its fixed!
```